addons/web/static/src/views/fields/badge_selection/badge_selection.scss

```diff
@@ -1,6 +1,12 @@
-.o_list_renderer .o_field_selection_badge {
-    .o_badge_border {
-        outline: 2px solid $o-enterprise-action-color;
-        outline-offset: -1px;
+.o_list_renderer {
+    .o_field_selection_badge {
+        .o_badge_border {
+            outline: 2px solid $o-enterprise-action-color;
+            outline-offset: -1px;
+        }
+    }
+
+    .o_field_state_selection {
```

Comment:
this file contains the style of the badge selection field :-)

You should move these rules into state_selection_field.scss